fe/lib/components/BrandSelect/styles.css.ts

```diff
@@ -0,0 +1,49 @@
+import { style } from '@vanilla-extract/css';
+import { calc } from '@vanilla-extract/css-utils';
+import { vars } from 'braid-design-system/css';
+
+const coverImageProps = style({
+  borderTopLeftRadius: vars.borderRadius.large,
+  borderTopRightRadius: vars.borderRadius.large,
+  height: calc.multiply(vars.grid, 40),
```

Comment:
Q: are our brand tiles too large? The AdCentre uses much smaller ones.

Reply:
hey @72636c can you send me the AdCentre link which you refer to? Is it https://talent.seek.com.au/account/branding/?  I see the tiles there are  almost similar size

Reply:
Ah, I meant the widget that's part of the job posting flow.

Reply:
ah, I see

Reply:
@72636c I've checked to resize the tiles, but it's responsive to the container, any suggestions to style it? and wonder how it looks when used in Ryanair Position Posting form

Reply:
All good Phong, I'm happy to leave as-is and we can always tweak it on later UX review. This is a big effort already without me scope creeping 😅